### PR TITLE
brought project over

### PR DESCRIPTION
**Pull request by kwm0304**
Finished this before reading how to submit projects, brought files over from other repo

### CODE REVIEW

**Comment by kalsson** [comment]
Hi @kwm0304 

I'm going to review your project.

- Issue 1: In your PrintInitialsPrompt() method, you want the user to add "Initials must be 3 letters.". How does the user know that?
image

 If I enter two digits the application crashes and throws an System.InvalidOperationException.
 
 `System.InvalidOperationException: Unbalanced markup stack. Did you forget to close a tag?
   at Spectre.Console.MarkupParser.Parse(String text, Style style) in /_/src/Spectre.Console/Internal/Text/Markup/MarkupParser.cs:line 55
   at Spectre.Console.AnsiConsoleExtensions.Markup(IAnsiConsole console, String value) in /_/src/Spectre.Console/Extensions/AnsiConsoleExtensions.Markup.cs:line 77
   at Spectre.Console.AnsiConsoleExtensions.MarkupLine(IAnsiConsole console, String value) in /_/src/Spectre.Console/Extensions/AnsiConsoleExtensions.Markup.cs:line 117
   at Spectre.Console.TextPrompt`1.<>c__DisplayClass63_0.<<ShowAsync>b__0>d.MoveNext() in /_/src/Spectre.Console/Prompts/TextPrompt.cs:line 178
--- End of stack trace from previous location ---
   at Spectre.Console.Internal.DefaultExclusivityMode.RunAsync[T](Func`1 func) in /_/src/Spectre.Console/Internal/DefaultExclusivityMode.cs:line 40
   at Spectre.Console.TextPrompt`1.ShowAsync(IAnsiConsole console, CancellationToken cancellationToken) in /_/src/Spectre.Console/Prompts/TextPrompt.cs:line 122
   at Spectre.Console.TextPrompt`1.Show(IAnsiConsole console) in /_/src/Spectre.Console/Prompts/TextPrompt.cs:line 111
   at Spectre.Console.AnsiConsole.Prompt[T](IPrompt`1 prompt) in /_/src/Spectre.Console/AnsiConsole.Prompt.cs:line 21
   at MathGame.CSA.Printer.PrintInitialsPrompt() in C:\Users\jonne\source\repos\CodeReviews.Console.MathGame\MathGame.kwm0304\Printer.cs:line 110
   at MathGame.CSA.GameLoop.GameOver() in C:\Users\jonne\source\repos\CodeReviews.Console.MathGame\MathGame.kwm0304\GameLoop.cs:line 68
   at MathGame.CSA.GameLoop.Run() in C:\Users\jonne\source\repos\CodeReviews.Console.MathGame\MathGame.kwm0304\GameLoop.cs:line 36
   at MathGame.CSA.GameLoop.DisplayMainMenu() in C:\Users\jonne\source\repos\CodeReviews.Console.MathGame\MathGame.kwm0304\GameLoop.cs:line 23
   at MathGame.CSA.Program.Main(String[] args) in C:\Users\jonne\source\repos\CodeReviews.Console.MathGame\MathGame.kwm0304\Program.cs:line 17`

File: MathGame.kwm0304/Printer.cs [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+ /  _____|    /   \     |   \/   | |   ____|    /  __  \  \   \  /   / |   ____||   _  \     
+|  |  __     /  ^  \    |  \  /  | |  |__      |  |  |  |  \   \/   /  |  |__   |  |_)  |    
+|  | |_ |   /  /_\  \   |  |\/|  | |   __|     |  |  |  |   \      /   |   __|  |      /     
+|  |__| |  /  _____  \  |  |  |  | |  |____    |  `--'  |    \    /    |  |____ |  |\  \----.
+ \______| /__/     \__\ |__|  |__| |_______|    \______/      \__/     |_______|| _| `._____|
+                                                                                             
+");
+    AnsiConsole.MarkupLine($"[blue]Score: {score}[/]");
+  }
+  public static string PrintInitialsPrompt()
```
Comment (kalsson) [inline MathGame.kwm0304/Printer.cs:108]: - Issue 1: In your PrintInitialsPrompt() method, you want the user to add "Initials must be 3 letters.". How does the user know that?
image

 If I enter two characters the application crashes and throws an System.InvalidOperationException. You don't handle if the user adds to little or to much initials.
 
```
System.InvalidOperationException: Unbalanced markup stack. Did you forget to close a tag?
   at Spectre.Console.MarkupParser.Parse(String text, Style style) in /_/src/Spectre.Console/Internal/Text/Markup/MarkupParser.cs:line 55
   at Spectre.Console.AnsiConsoleExtensions.Markup(IAnsiConsole console, String value) in /_/src/Spectre.Console/Extensions/AnsiConsoleExtensions.Markup.cs:line 77
   at Spectre.Console.AnsiConsoleExtensions.MarkupLine(IAnsiConsole console, String value) in /_/src/Spectre.Console/Extensions/AnsiConsoleExtensions.Markup.cs:line 117
   at Spectre.Console.TextPrompt`1.<>c__DisplayClass63_0.<<ShowAsync>b__0>d.MoveNext() in /_/src/Spectre.Console/Prompts/TextPrompt.cs:line 178
--- End of stack trace from previous location ---
   at Spectre.Console.Internal.DefaultExclusivityMode.RunAsync[T](Func`1 func) in /_/src/Spectre.Console/Internal/DefaultExclusivityMode.cs:line 40
   at Spectre.Console.TextPrompt`1.ShowAsync(IAnsiConsole console, CancellationToken cancellationToken) in /_/src/Spectre.Console/Prompts/TextPrompt.cs:line 122
   at Spectre.Console.TextPrompt`1.Show(IAnsiConsole console) in /_/src/Spectre.Console/Prompts/TextPrompt.cs:line 111
   at Spectre.Console.AnsiConsole.Prompt[T](IPrompt`1 prompt) in /_/src/Spectre.Console/AnsiConsole.Prompt.cs:line 21
   at MathGame.CSA.Printer.PrintInitialsPrompt() in C:\Users\jonne\source\repos\CodeReviews.Console.MathGame\MathGame.kwm0304\Printer.cs:line 110
   at MathGame.CSA.GameLoop.GameOver() in C:\Users\jonne\source\repos\CodeReviews.Console.MathGame\MathGame.kwm0304\GameLoop.cs:line 68
   at MathGame.CSA.GameLoop.Run() in C:\Users\jonne\source\repos\CodeReviews.Console.MathGame\MathGame.kwm0304\GameLoop.cs:line 36
   at MathGame.CSA.GameLoop.DisplayMainMenu() in C:\Users\jonne\source\repos\CodeReviews.Console.MathGame\MathGame.kwm0304\GameLoop.cs:line 23
   at MathGame.CSA.Program.Main(String[] args) in C:\Users\jonne\source\repos\CodeReviews.Console.MathGame\MathGame.kwm0304\Program.cs:line 17
```

**Review by kalsson — CHANGES_REQUESTED** [review]

**Review by kalsson — CHANGES_REQUESTED** [review]
- Issue 2: If I choose more than one question to answer and I enter the wrong answer to the first question, then the application does not give me the rest of the questions, I am being sent to enter my initials.

**Comment by kwm0304** [comment]
Thanks for looking over it, I'll fix those issues and resubmit

**Comment by kwm0304** [comment]
> * Issue 2: If I choose more than one question to answer and I enter the wrong answer to the first question, then the application does not give me the rest of the questions, I am being sent to enter my initials.

I had the game ending on a missed question originally and just setting Completed to false, but it does make more sense to let the user finish out the game

**Comment by kalsson** [comment]
> Thanks for looking over it, I'll fix those issues and resubmit

No problem. Remember not to resubmit this project via The C# Academy if that's what you mean, you only need to make your changes on your computer and then commit to the same branch. I'll get a message when you've done that and then I'll choose to pull the latest changes from your (main branch). 😉😁

**Review by kalsson — COMMENTED** [review]
👍

**Pull request merged**